public/app/features/dashboard/services/DashboardSrv.ts

```diff
@@ -51,17 +48,13 @@ export class DashboardSrv {
     let promise;
 
     if (isStarred) {
-      promise = promiseToDigest(this.$rootScope)(
-        backendSrv.delete('/api/user/stars/dashboard/' + dashboardId).then(() => {
-          return false;
-        })
-      );
+      promise = backendSrv.delete('/api/user/stars/dashboard/' + dashboardId).then(() => {
```

Comment:
You should probably use `getBackendSrv()` instead to get the singleton instance. No need to create new instances :)

Reply:
naw, i don't think it creates new instances. it's just a different way of importing the same instance that `getBackendSrv()` returns. i'll switch it to `getBackendSrv` though 👍